frontend/yarn.lock

```diff
@@ -2413,6 +2413,11 @@
   resolved "https://registry.yarnpkg.com/@popperjs/core/-/core-2.11.5.tgz#db5a11bf66bdab39569719555b0f76e138d7bd64"
   integrity sha512-9X2obfABZuDVLCgPK9aX0a/x4jaOEweTTWE2+9sr0Qqqevj2Uv5XorvusThmc9XGYpS9yI+fhh8RTafBtGposw==
 
+"@react-oauth/google@^0.5.1":
```

Comment:
Are we pushing yarn.lock files to github? I think bowen had some dependancy changes but didn't push the yarn.lock file. If we're putting it on the repo, then I think we should add his yarn.lock file before merging his pr too

Reply:
Yeah don't see any reason not to